src/loader/aarch64/pe/mod.rs

```diff
@@ -168,6 +180,40 @@ impl KernelLoader for PE {
     }
 }
 
+/// load uefi for aarch64
```

Comment:
nit: can you format the comment similar to how other comments are formatted: `Load UEFI image in guest memory`.

Reply:
Ok, thanks